metadata.json

```diff
@@ -22,7 +22,7 @@
     },
     {
       "name": "puppetlabs/mongodb",
-      "version_requirement": ">= 0.10.0 < 1.0.0"
+      "version_requirement": ">= 0.17.0 < 1.0.0"
```

Comment:
Given this commit hasn't made it into a release, should we wait for that and depend on it?